doc/classes/CPUParticles2D.xml

```diff
@@ -68,132 +93,194 @@
 		<member name="emission_points" type="PoolVector2Array" setter="set_emission_points" getter="get_emission_points">
 		</member>
 		<member name="emission_rect_extents" type="Vector2" setter="set_emission_rect_extents" getter="get_emission_rect_extents">
+			The rectangle's extents if [member emission_shape] is set to [const EMISSION_SHAPE_RECTANGLE].
 		</member>
 		<member name="emission_shape" type="int" setter="set_emission_shape" getter="get_emission_shape" enum="CPUParticles2D.EmissionShape">
+			Particles will be emitted inside this region. Use [enum EmissionShape] for values. Default value: [constant EMISSION_SHAPE_POINT].
 		</member>
 		<member name="emission_sphere_radius" type="float" setter="set_emission_sphere_radius" getter="get_emission_sphere_radius">
+			The circle's radius if [member emission_shape] is set to [const EMISSION_SHAPE_CIRCLE].
 		</member>
 		<member name="emitting" type="bool" setter="set_emitting" getter="is_emitting">
+			If [code]true[/code], particles are being emitted. Default value: [code]true[/code].
 		</member>
 		<member name="explosiveness" type="float" setter="set_explosiveness_ratio" getter="get_explosiveness_ratio">
+			ow rapidly particles in an emission cycle are emitted. If greater than [code]0[/code], there will be a gap in emissions before the next cycle begins. Default value: [code]0[/code].
 		</member>
 		<member name="fixed_fps" type="int" setter="set_fixed_fps" getter="get_fixed_fps">
+			The particle system's frame rate is fixed to a value. For instance, changing the value to 2 will make the particles render at 2 frames per second. Note this does not slow down the simulation of the particle system itself.
 		</member>
 		<member name="flag_align_y" type="bool" setter="set_particle_flag" getter="get_particle_flag">
+			Align y-axis of particle with the direction of its velocity.
 		</member>
 		<member name="flatness" type="float" setter="set_flatness" getter="get_flatness">
```

Comment:
This one was missed. It's documented in `CPUParticles` so can likely be adapted from there.

Reply:
I did a quick test of this one and it's not clear it's working as expected. I'll hold off on the description until I can verify if it needs to be added to #29692